sql/core/src/test/resources/sql-tests/inputs/postgreSQL/comments.sql

```diff
@@ -26,23 +26,23 @@ SELECT 'after multi-line' AS fifth;
 SELECT 'trailing' as x1; -- inside block comment
 */
 
-/* This block comment surrounds a query which itself has a block comment...
-SELECT /* embedded single line */ 'embedded' AS x2;
-*/
+-- /* This block comment surrounds a query which itself has a block comment...
+-- SELECT /* embedded single line */ 'embedded' AS x2;
+-- */
 
```

Comment:
> Spark SQL can't support nested bracketed comments and I will open another PR to support it.

We need to update this file in this PR?  If you will work on that, I think its ok to update this file in the next one.

Reply:
Thanks for your review. I have a question the nested bracketed comments will throw parsed exception not look good. Should I display the parsed exception into output?

Reply:
Yea, I think better error messages look good if we can fix it easily.

Reply:
It won't be easy for the time being. So I want comment out temporarily.

Reply:
@maropu I think we can fix the test cases in 3.0. Regarding https://github.com/apache/spark/pull/27495, it is an enhancement, we can merge it to master only.

Reply:
We need this change to comment out these tests in branch-3.0? If branch-3.0 doesn't support these nested comments,  its better to fix them so that the test could throw an exception for nested comments here?

Reply:
OK. I will not comment out nested comments and throw exception into golden files.